fe1-web/src/features/wallet/screens/WalletCreateSeed.tsx

```diff
@@ -83,75 +77,72 @@ const WalletCreateSeed = () => {
     }
   };
 
+  const toolbarItems: ToolbarItem[] = [
+    {
+      title: STRINGS.wallet_welcome_button_restore_seed,
+      onPress: () => navigation.navigate(STRINGS.navigation_app_wallet_insert_seed),
+      buttonStyle: 'secondary',
+    },
+    {
+      id: 'exploring_selector',
+      title: STRINGS.wallet_welcome_button_start_exploring,
+      onPress: connectWithSeed,
+    },
+  ];
+
   return (
-    <View style={styles.welcomeView}>
-      <ScreenWrapper>
-        <View style={containerStyles.centeredY}>
-          <Text style={[Typography.heading, Typography.negative]}>
-            {STRINGS.wallet_welcome_heading}
+    <ScreenWrapper toolbarItems={toolbarItems}>
+      <View style={containerStyles.centeredY}>
+        <Text style={Typography.heading}>{STRINGS.wallet_welcome_heading}</Text>
+        <Text style={Typography.paragraph}>{STRINGS.wallet_welcome_text_first_time}</Text>
+        <Text style={Typography.paragraph}>
+          <Text>{STRINGS.wallet_welcome_text_wallet_explanation_1}</Text>
+          <Text> </Text>
```

Comment:
does it make sense to have all these empty `<Text>` fields ? 
Is there no better way ?

Reply:
Well either that or have sentences start or end with a space which is bad too 😅

Reply:
Or, of course, rewrite the sentences / combine them as suggested above. The reason for these artifacts are the differently colored text sections. Or do you see a another option?